src/test/java/ControllerTest/RussianCheckersControllerTest.java

```diff
@@ -75,13 +76,14 @@ public void testStartingPosition() {
 
     @Test
     public void testLegalFirstMoves() {
-        assertTrue(controller.isMoveLegal(1, 5, 0, 4));
-        assertTrue(controller.isMoveLegal(1, 5, 2, 4));
-        assertTrue(controller.isMoveLegal(3, 5, 2, 4));
-        assertTrue(controller.isMoveLegal(3, 5, 4, 4));
-        assertTrue(controller.isMoveLegal(5, 5, 4, 4));
-        assertTrue(controller.isMoveLegal(5, 5, 6, 4));
-        assertTrue(controller.isMoveLegal(7, 5, 6, 4));
+        board.addToPossibleMoves();
```

Comment:
Jeżeli to jest konieczne, to czy można to jakoś wepchać do środka, żeby się robiło automatycznie, albo w initialize()? Tak samo w pozostałych klasach testowych.